x-pack/plugins/apm/server/lib/transactions/fetcher/index.ts

```diff
@@ -39,9 +39,9 @@ interface TopTraceOptions {
 
 export type Options = TopTransactionOptions | TopTraceOptions;
 
-export type ESResponse = PromiseReturnType<typeof transactionGroupsFetcher>;
+export type ESResponse = PromiseReturnType<typeof transactionFetcher>;
```

Comment:
I'm not sure all of this should be renamed tbh. There is a difference between transactions and transaction groups